test/CMakeLists.txt

```diff
@@ -25,28 +25,33 @@ set( UNITY_DIR ${UNIT_TEST_DIR}/Unity CACHE INTERNAL "Unity library source direc
 option( UNITTEST
         "Set this to ON to build unit tests. This will clone the required Unity test framework submodule if it is not cloned already."
         OFF )
+option( COV_ANALYSIS
+        "Set this to ON to build coverity analysis project."
+        ON )
```

Comment:
Why not have this default to `OFF`? Not all users will have coverity licenses.

And if you really want this be defaulted to `ON` then I suggest the help_text be updated to "Set this to OFF to disable coverity analysis build configurations."

Reply:
Thanks for the suggestion. Update in new commit.